libs/feature/map/src/lib/utils/map-utils.service.ts

```diff
@@ -159,17 +159,31 @@ export class MapUtilsService {
   }
 
   getWmtsOptionsFromCapabilities(link: MetadataLink): Observable<Options> {
+    const getCapabilitiesUrl = new URL(link.url, window.location.toString())
+    getCapabilitiesUrl.searchParams.set('SERVICE', 'WMTS')
+    getCapabilitiesUrl.searchParams.set('REQUEST', 'GetCapabilities')
     return from(
-      fetch(link.url)
-        .then(function (response) {
+      fetch(getCapabilitiesUrl.toString())
+        .then(async function (response) {
+          if (!response.ok) {
+            throw new Error(`WMTS GetCapabilities HTTP request failed with code ${
+              response.status
+            } and body:
+${await response.text()}`)
```

Comment:
Weird formatting... also noticing that we haven't translated any error messages yet.

Reply:
it looks weird but the result makes sense since we're in a template string